src/main/resources/models/tables/PlayerCountryView.hjson

```diff
@@ -0,0 +1,20 @@
+{
+    tables: [{
+        name: PlayerCountryView
```

Comment:
Lets change the name to PlayerCountry here and in the file name to match the table name itself.